code/modules/mob/living/simple_animal/emote.dm

```diff
@@ -0,0 +1,5 @@
+/datum/emote/simple_animal/scream
```

Comment:
This literally only exists because of things calling `M.emote("scream")` doesn't it?

Reply:
Simple animals were able to scream before this PR I'm pretty sure